app/views/helpers/Formatters.scala

```diff
@@ -30,9 +30,13 @@ trait DateFormatters {
 }
 
 trait FileFormatters {
+
+  private val kbMin = 1000
+  private val mbMin = 1000000
+
   def fileSize(size: Long): String = size match {
-    case kb if 1000 until 1000000 contains kb => s"${kb / 1000}KB"
```

Comment:
**a type was inferred to be `AnyVal`; this may indicate a programming error** Instead of until, may be try with < and > symbols to check the condition